profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.module

```diff
@@ -486,3 +491,25 @@ function _nexteuropa_varnish_temporary_message() {
 
   return FALSE;
 }
+
+/**
+ * Trims the base path from the given path.
+ *
+ * @param string $path
+ *   Path which is going to be trimmed.
+ *
+ * @return string
+ *   Trimmed path.
+ */
+function _nexteuropa_varnish_trim_base_path($path) {
+  $base_path = base_path();
+  $base_path_position = strpos($path, $base_path);
+
+  if ($base_path_position !== FALSE) {
+    $trimmed_path = substr_replace($path, '', $base_path_position, strlen($base_path));
+
+    return "/" . $trimmed_path;
```

Comment:
I would only add the leading '/' if it is not present yet; testing value before modifying it is required here IMO.